conans/client/source.py

```diff
@@ -105,7 +105,7 @@ def get_sources_from_exports():
                 # First of all get the exported scm sources (if auto) or clone (if fixed)
                 _run_cache_scm(conanfile, scm_sources_folder, output)
                 # so self exported files have precedence over python_requires ones
-                merge_directories(export_folder, conanfile.source_folder)
+                merge_directories(export_folder, conanfile.folders.base_source)
```

Comment:
I am confused that the ``export_folder`` went to source_folder, and instead below ``export_source_folder`` was already targeted to ``folders.base_source``? I expected the opposite. I guess that ``export_source_folder`` is already containing the "source" layout, is it that?

Reply:
Yes, the  `export_source_folder` is containing the `src/` so it goes to the base.